.github/workflows/plutus-tx-template.yml

```diff
@@ -17,40 +17,40 @@ jobs:
     steps:
       - name: Checkout plutus-tx-template Repo
         uses: actions/checkout@main
-        with: 
+        with:
           repository: IntersectMBO/plutus-tx-template
-          path: plutus-tx-template 
+          path: plutus-tx-template
 
       - name: Checkout plutus Repo
         uses: actions/checkout@main
-        with: 
+        with:
           path: plutus-tx-template/plutus
-  
+
       - name: Overwrite cabal.project.local
         uses: DamianReeves/write-file-action@master
         with:
           path: plutus-tx-template/cabal.project.local
           write-mode: overwrite
           contents: |
-            packages: 
+            packages:
               plutus/plutus-tx
               plutus/plutus-tx-plugin
               plutus/plutus-core
               plutus/plutus-ledger-api
               plutus/prettyprinter-configurable
-            
+
             allow-newer:
                 plutus-tx
               , plutus-tx-plugin
               , plutus-core
-              , plutus-ledger-api 
+              , plutus-ledger-api
               , prettyprinter-configurable
-          
+
       - name: Build Project With Docker
         run: |
-          cd plutus-tx-template 
+          cd plutus-tx-template
           docker run \
             -v ./.:/workspaces/plutus-tx-template \
             -w /workspaces/plutus-tx-template \
             -i ghcr.io/input-output-hk/devx-devcontainer:x86_64-linux.ghc96-iog \
-            bash -ic "cabal update && cabal run plutus-tx-template && test -e validator.uplc"
+            bash -ic "cabal update && cabal build all"
```

Comment:
We don't want to run any test?

Reply:
There's no test to run at the moment. The old command simply runs the executable, and checks that the output file exists, which isn't testing much, and now it no longer works.

Reply:
Thanks!